input/pagecontent/5_content-sharing.md

```diff
@@ -0,0 +1,78 @@
+
+FHIR resources are used to convey the structured information being exchanged in `[FHIR resource]-update` operations.  However, it is possible that these resources are never persisted in a FHIR server.  During the exchange of information, the content (FHIR resource instances) is often very dynamic in nature with a user creating, modifying, and even removing information which is being exchanged.  For example, a measurement made in an imaging application could be altered many times before it is finalized and it could be removed.
+
+### Responsibilities of a FHIRcast Hub and a Subscribed Client
+
+If supporting content sharing, a FHIRcast Hub MUST fulfill additional responsibilities:
+1. Assign and maintain an anchor context's `versionId` when processing a `[FHIR resource]-open` request - the `versionId` does not need to follow any convension but must unique in the scope of a topic  
+2. Validate `[FHIR resource]-update` and `[FHIR resource]-select` requests for the correct `versionId` and reject the request if the version does not match the current `versionId` by returning a 4xx/5xx HTTP Status Code rather than updating the content or indication of selection
+3. Assign and maintain a new `versionId` for the anchor context's content and provide the new `versionId` along with the `priorVersionId` in the event corresponding to the validated update request
+4. Maintain a list of current FHIR resource content in the anchor context so that it may provide the anchor context's content in response to a [`GET Context`](3-16-get-context.html) request
+5. When a `[FHIR resource]-close` request is received, the Hub should dispose of the content for the anchor context since the Hub has no responsibility to persist the content
+
+A Hub is not responsible for structurally validating FHIR resources.  While a Hub must be able to successfully parse FHIR resources sufficiently to perform its required capabilities (e.g., find the `id` of a resource), a Hub is not responsible for additional structural checking. 
+
+A Hub is not responsible for any long-term persistence of shared information and should purge the content when a `[FHIR resource]-close` request is received.
+
+Additionally, a Hub is not responsible to prevent applications participating in exchanging structured information from causing inconsistencies in the information exchanged.  For example, an inconsistency could arise if an application removes from the anchor context's content a resource which is referenced by another resource.  The Hub may check `[FHIR resource]-update` requests for such inconsistencies and reject the request with an appropriate error message; however, it is not required to do so.  Additionally, a Hub may check for inconsistencies which it deems to be critical but not perform exhaustive validation. For example, a Hub could validate that the content in a `DiagnosticReport` anchor context always includes at least one primary imaging study.
```

Comment:
In general, but also specifically this paragraph does not use enough normative language to clearly articulate exactly the behavior required by a client and a hub.

Reply:
OK...I guess I can work on getting more normative.  The text was originally in a section I had denoted as informative.  Let me work on this so it is more "requirement'ish"